crates/sui-replay-2/src/lib.rs

```diff
@@ -188,20 +228,98 @@ impl FromStr for Node {
     }
 }
 
+/// Load replay configuration from ~/.sui/sui_config/replay.toml file.
+/// Returns default config (all fields set to None) if file cannot be found or read.
+pub fn load_config_file() -> anyhow::Result<ReplayConfigStable> {
+    let config_dir = match sui_config_dir() {
+        Ok(dir) => dir,
+        Err(e) => {
+            eprintln!("Cannot locate replay config file: {e}");
```

Comment:
Should we print this if the file isn't present? It seems like it could be annoying unless we expect that everyone that runs a replay command will have a config which feels a bit overzealous/annoying.

Reply:
This is actually printed when Sui config dir cannot be found (`~/.sui/sui_config`) and not when replay config cannot be found (this may be because OS call to obtain current directory fails or the Sui config dir cannot be created). I just made the error message a bit more related to replay (rather than saying something like "Sui config directory cannot be located")